cli/src/tasks/utils.ts

```diff
@@ -10,3 +10,13 @@ export function formatBytes(bytes: number): string {
   const i = Math.floor(Math.log(bytes) / Math.log(k))
   return parseFloat((bytes / Math.pow(k, i)).toFixed(2)) + ' ' + sizes[i]
 }
+
+/**
+ * Returns the maximum of two bigints
+ * @param a - the first bigint
+ * @param b - the second bigint
+ * @returns the maximum of the two bigints
+ */
+export function maxBigInt(a: bigint, b: bigint): bigint {
+  return a > b ? a : b
+}
```

Comment:
```suggestion
export const maxBigInt = (a: bigint, b: bigint): bigint => (a > b ? a : b);
```
little shorter

Reply:
I don't see the point of changing a named function to an arrow function in this case
That won't change anything on the performance side, just reduce a bit the readability and break the consistency with the "coding style" (by that, I just mean the fact we used named functions everywhere else)